api/services/tasks/v1/tasks.proto

```diff
@@ -51,6 +51,14 @@ service Tasks {
 	rpc Metrics(MetricsRequest) returns (MetricsResponse);
 
 	rpc Wait(WaitRequest) returns (WaitResponse);
+
+	rpc CreatePageServer(CreatePageServerRequest) returns (CreatePageServerResponse);
```

Comment:
We are going to stabilize the interface of the `task` service in v1.0, but we are unlikely to stabilize the phaul interfaces in v1.0, so please split these methods to a new service?